### PR TITLE
React person task

### PR DESCRIPTION
**Pull request by Kavster92**
[DEMO LINK](https://Kavster92.github.io/react_person/)

### CODE REVIEW

**Comment by dolnaro** [comment]
Your demo link is not working.

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,31 @@
...
+      ? 'wife'
+      : 'husband'
+  );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+      <p className="Person__partner">
+        {
```
Comment (dolnaro) [inline src/components/Person/Person.jsx:23]: You might want to edit these lines to make them easier to read.

**Review by dolnaro — CHANGES_REQUESTED** [review]

File: src/components/Person/Person.jsx (hunk 1 above)
Comment (Kavster92) [reply src/components/Person/Person.jsx:23]: done bro

Comment (dolnaro) [reply src/components/Person/Person.jsx:23]: Did you push and deploy? Because I can't see it yet.

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,31 @@
...
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerOfAPerson = (
+    sex === 'm'
+      ? 'wife'
+      : 'husband'
+  );
```
Comment (naziamoff) [inline src/components/Person/Person.jsx:16]: let's simplify it a bit
```suggestion
  const partner = sex === 'm'
      ? 'wife'
      : 'husband';
```

File: src/components/Person/Person.jsx (hunk 1 above)
Comment (naziamoff) [reply src/components/Person/Person.jsx:23]: yeah, I don't see it either

**Review by naziamoff — CHANGES_REQUESTED** [review]
Good job, but seems you did not push after fixing the comments of @dolnaro

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,32 @@
...
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+      <p className="Person__partner">
+        {
+          isMarried
+            ? `${partnerName} is my ${partnerOfAPerson}`
+            : 'I am not married'
+        }
```
Comment (naziamoff) [inline src/components/Person/Person.jsx:28]:
```suggestion
        {isMarried 
          ? `${partnerName} is my ${partnerOfAPerson}`
          : 'I am not married'}
```

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,32 @@
...
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
```
Comment (naziamoff) [inline src/components/Person/Person.jsx:22]: should be moved to the new line
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
       )}
```

**Review by naziamoff — APPROVED** [review]

**Pull request closed without merging**